site/assets/js/search.js

```diff
@@ -5,51 +5,38 @@
 (() => {
   'use strict'
 
-  const inputElement = document.getElementById('search-input')
-
-  if (!window.docsearch || !inputElement) {
+  if (!window.docsearch) {
```

Comment:
Any reason why we don't still check for `#docsearch`? We need the search to be present too. I suggest we query the element like before here and reuse it later as we did.

Reply:
We don't really need to reuse it as much as before... maybe in the `container` option further down we could do that, but doesn't feel necessary to me.